package.json

```diff
@@ -59,18 +59,37 @@
     "@babel/polyfill": "^7.4.4",
     "@babel/preset-env": "^7.16.0",
     "@babel/preset-react": "^7.16.0",
-    "@babel/preset-typescript": "^7.16.0",
+    "@babel/preset-typescript": "^7.18.6",
     "@babel/register": "^7.4.0",
     "@emotion/babel-preset-css-prop": "^10.0.9",
+    "@storybook/addon-actions": "^6.5.10",
+    "@storybook/addon-essentials": "^6.5.10",
+    "@storybook/addon-interactions": "^6.5.10",
+    "@storybook/addon-knobs": "^6.4.0",
+    "@storybook/addon-links": "^6.5.10",
+    "@storybook/builder-webpack4": "^6.5.10",
+    "@storybook/manager-webpack4": "^6.5.10",
+    "@storybook/react": "^6.5.10",
+    "@storybook/testing-library": "^0.0.13",
+    "@svgr/webpack": "^6.3.1",
     "babel-loader": "^8.0.5",
     "babel-plugin-module-resolver": "^3.2.0",
     "babel-preset-env": "^1.7.0",
-    "ts-node-dev": "^1.1.8",
-    "tsc-watch": "^4.5.0",
-    "typescript": "^3.9.5",
+    "husky": "^2.2.0",
     "plop": "^2.3.0",
     "prettier": "^2.0.5",
     "pretty-quick": "^2.0.1",
-    "husky": "^2.2.0"
+    "react": "^18.2.0",
+    "react-docgen-typescript-loader": "^3.7.2",
+    "react-dom": "^18.2.0",
+    "react-json-view": "^1.21.3",
+    "ts-node-dev": "^1.1.8",
+    "ttypescript": "^1.5.13",
+    "typescript": "^3.9.5"
+  },
+  "prettier": {
+    "printWidth": 100,
```

Comment:
I politely object to the 100 print width - 80 is the default and not a hard limit, and a bit more usable with 2 files open side by side

Reply:
While I don't have a strong opinion about this one (I'd go for 123, really), I'm fairly certain all our apps use 100 as a standard.
Y'all decide whether we should change to 80 everywhere, for consistency.

Reply:
discussed on slack, carry on with 100